examples/sandbox/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shapeshiftoss/hdwallet-sandbox",
-  "version": "1.50.9",
+  "version": "1.50.10-alpha.1",
```

Comment:
Run `lerna:version` once stamped and before merging this

Reply:
https://github.com/shapeshift/hdwallet/pull/629/commits/c75ab12c9c0130ca63fd79959b4e69ed16eed1e5